cumulus/parachains/integration-tests/emulated/assets/asset-hub-kusama/src/tests/send.rs

```diff
@@ -150,8 +150,9 @@ fn send_xcm_from_para_to_system_para_paying_fee_with_assets_works() {
 
 	let origin_kind = OriginKind::SovereignAccount;
 	let fee_amount = ASSET_MIN_BALANCE * 1000000;
-	let native_asset =
-		(X2(PalletInstance(ASSETS_PALLET_ID), GeneralIndex(ASSET_ID.into())), fee_amount).into();
+	let junctions: Junctions =
+		[PalletInstance(ASSETS_PALLET_ID), GeneralIndex(ASSET_ID.into())].into();
+	let native_asset = (junctions, fee_amount).into();
```

Comment:
Why aren't you able to replace `junctions` with just the value, i.e. `[PalletInstance(ASSETS_PALLET_ID), GeneralIndex(ASSET_ID.into())]`, without calling `.into()`? This pattern used to work before...

Reply:
This does indeed work, seems I was overly cautious